database/fake/fakeData.js

```diff
@@ -0,0 +1,40 @@
+const faker = require('faker');
+const mysql = require('mysql');
+
+// add more commerce.productAdjective to this string-object?
+const fakeProduct = () => {
+  const newData = faker.fake('{{commerce.color}}, {{commerce.department}}, {{commerce.productName}}, {{commerce.price}}, {{commerce.productAdjective}}, {{commerce.productMaterial}}, {{commerce.product}}, {{image.image}}');
```

Comment:
I think you should break the line into several lines to make it more readable.

Reply:
Good call. I'll fix that up.

Reply:
Ah, it seems like vs code isn't gonna let me split them up. Even '\' doesn't stop string escape at the end of a line.